roadmap/implementers-guide/src/node/utility/network-bridge.md

```diff
@@ -44,3 +49,12 @@ On `ReportPeer` message:
 On `SendMessage` message:
 
 - Issue a corresponding `ProtocolMessage` to each listed peer with given protocol ID and bytes.
+
+[NBM]: ../../types/overseer-protocol.md#network-bridge-message
+
+On `ConnectToValidators` message:
+
+- Determine the DHT keys to use for each validator based on the relay-chain state and Runtime API.
+- Recover the Peer IDs of the validators from the DHT. There may be more than one peer ID per validator.
+- Accumulate all `(ValidatorId, PeerId)` pairs and send on the response channel.
+- Feed all Peer IDs to the discovery utility the underlying network provides.
```

Comment:
What is meant with `discovery utility` here? Would the line below work as well?

```suggestion
- Add one `PeerId` per validator as a priority group to the `PeerSet`.
```

Reply:
I guess so, but I talked with Pierre and we wanted to do away with this priority groups thing. The previous line seems more general and can be adapted based on what is actually done.

Reply:
The guide in general leans away from implementation details of Substrate. I believe priority groups are such a detail.

Reply:
```suggestion
- Feed all Peer IDs to the peer set manager the underlying network provides.
```

In case we want to keep it generic I would suggest the following. I don't think `discovery utility` is the right term here as the peer has already been discovered at this point.

Reply:
addressed in #1535